### PR TITLE
Update cli and add geo filter after CDN RP version change

### PR DESCRIPTION
**Pull request by csmengwan**
The content to be added to Changelog is as follows:
* 
* 
* 
update cdn pacakge version

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@csmengwan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (qinwa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 1]
```diff
@@ -1422,4 +1429,334 @@ exports.init = function(cli) {
...
+  //Add geo filter command
+  geoFilter.command('add [relative-path] [endpoint-name] [profile-name] [resource-group] [action] [country-codes]')
+    .description($('Add a geo filters by endpoint name, profile name, and resource group'))
+    .usage('[options] <relative-path> <endpoint-name> <profile-name> <resource-group> <action> <country-codes>')
+	  .option('-r, --relative-path <relative-path>', $('Relative path of the geo filter'))
+    .option('-e, --endpoint-name <endpoint-name>', $('Name of the endpoint'))
+    .option('-p, --profile-name <profile-name>', $('Name of the profile'))
+    .option('-g, --resource-group <resource-group>', $('The resource group of the Azure Cdn Profile'))
+	  .option('-a, --action <action>', $('Action of the geo filter. Valid input: -a [Block|Allow]'))
+	  .option('-c, --country-codes <country-codes>', $('List of country codes to apply the geo filter'))
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: - Do you expect 2 letter or 3 letter country codes?
- Should they be comma separated or semicolon separated?
- Can the user provide an empty list? Please specify that in the description whether it is allowed or not? If allowed then provide an example of how can this be achieved?

 Please clarify the above points in the description and provide examples for the same.

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 2]
```diff
@@ -1422,4 +1429,334 @@ exports.init = function(cli) {
...
+          });
+        }
+      });
+    });
+
+  //Add geo filter command
+  geoFilter.command('add [relative-path] [endpoint-name] [profile-name] [resource-group] [action] [country-codes]')
+    .description($('Add a geo filters by endpoint name, profile name, and resource group'))
+    .usage('[options] <relative-path> <endpoint-name> <profile-name> <resource-group> <action> <country-codes>')
+	  .option('-r, --relative-path <relative-path>', $('Relative path of the geo filter'))
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: please provide an example for relative-path.

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 3]
```diff
@@ -1422,4 +1429,334 @@ exports.init = function(cli) {
...
+  //Update geo filter command
+  geoFilter.command('set [relative-path] [endpoint-name] [profile-name] [resource-group]')
+    .description($('Update a geo filters by endpoint name, profile name, and resource group'))
+    .usage('[options] <relative-path> <endpoint-name> <profile-name> <resource-group> [action] [country-codes]')
+	  .option('-r, --relative-path <relative-path>', $('Relative path of the geo filter'))
+    .option('-e, --endpoint-name <endpoint-name>', $('Name of the endpoint'))
+    .option('-p, --profile-name <profile-name>', $('Name of the profile'))
+    .option('-g, --resource-group <resource-group>', $('The resource group of the Azure Cdn Profile'))
+	  .option('-a, --action [action]', $('Action of the geo filter. Block or Allow.'))
+	  .option('-c, --country-codes [country-codes]', $('List of country codes to apply the geo filter'))
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: indentation and make sure to update the descriptions over here as well.

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 4]
```diff
@@ -1422,4 +1429,334 @@ exports.init = function(cli) {
...
+    .option('--subscription <subscription>', $('the subscription identifier'))
+    .execute(function(relativePath, endpointName, profileName, resourceGroup, options, _) {
+      ///////////////////////
+      // Parse arguments.  //
+      ///////////////////////
+
+      relativePath = cli.interaction.promptIfNotGiven($('Relative path: '), relativePath, _);
+      endpointName = cli.interaction.promptIfNotGiven($('Endpoint name: '), endpointName, _);
+      profileName = cli.interaction.promptIfNotGiven($('Profile name: '), profileName, _);	  
+	    resourceGroup = cli.interaction.promptIfNotGiven($('Resource group name: '), resourceGroup, _);
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: indentation

**Comment by amarzavery** [comment]
@csmengwan - Please provide accurate content in the PR description. This helps us in providing accurate content in the changelog.

**Pull request merged**